args_test.go

```diff
@@ -232,3 +250,48 @@ func testGetCleanupTimeWithError(t *testing.T) {
 		require.Error(t, err)
 	}
 }
+
+// TestGetDockerConfigs tests that we can correctly get the docker config input parameters
+func TestGetDockerConfigs(t *testing.T) {
+	t.Run("NoError", testGetDockerConfigsNoError)
+	t.Run("Empty", testGetDockerConfigsEmpty)
+	t.Run("WithError", testGetDockerConfigsWithError)
+}
+
+// testGetDockerConfigsNoError tests that the correctly formated input can be parsed without error
+func testGetDockerConfigsNoError(t *testing.T) {
+	defer viper.Reset()
+
+	viper.Set(ContainerImageNameKey, testContainerImageName)
+	viper.Set(ContainerImageIDKey, testContainerImageID)
+	viper.Set(ContainerLabelsKey, testContainerLabels)
+	viper.Set(ContainerEnvKey, testContainerEnv)
+
+	args, err := getDockerConfigs()
+	require.NoError(t, err)
+	assert.Equal(t, args.ContainerImageName, testContainerImageName)
+	assert.Equal(t, args.ContainerImageID, testContainerImageID)
```

Comment:
Minor: let's flip the inputs to `assert.Equal`. `testContainerImage{Name,ID}` should be the expected value, not the actual value.

Reply:
Makes sense, will udpate